oak_restricted_kernel_launcher/README.md

```diff
@@ -16,20 +16,17 @@ must be built.
 
 (instructions gained from inspecting xtask, may change in the future)
 
-Stage0, the restricted kernel, and an enclave app may be built like so:
-
 ```shell
-just stage0_bin oak_restricted_kernel_wrapper oak_echo_raw_enclave_app
-```
+# Stage0, the restricted kernel, and an enclave app may be built like so:
```

Comment:
(nit) The previous format seems easier to parse and copy paste to me

Reply:
I found myself always wanting to build and run at the same time, so new changes are reflected. Esp since builds are cached. Hence one command is nice imo.